apps/api/src/.env.development

```diff
@@ -33,3 +33,6 @@ VERCEL_BASE_URL=https://api.vercel.com
 FF_IS_TOPIC_NOTIFICATION_ENABLED=true
 
 STORE_NOTIFICATION_CONTENT=
+
+MAX_NOVU_INTEGRATIONS=300
```

Comment:
```suggestion
MAX_NOVU_INTEGRATION_MAIL_REQUESTS=300
```

Reply:
I think it might be better indicative of the usage of this variable. Wdyt?

Reply:
yes